components/brave_new_tab_ui/components/default/gridSites/index.ts

```diff
@@ -202,7 +234,7 @@ interface TileProps {
   isMenuShowing: boolean
 }
 
-export const Tile = styled('a')<TileProps>`
+export const Tile = styled('a') <TileProps>`
```

Comment:
I don't like this space but my formatter does it by default. Is there a lint file I can get VSCode to use?

Reply:
hmm, I do not get this locally. I don't think we have a format file - just eslint and we _do_ need to work on our frontend languages format config...
I personally have the VSCode eslint plugin with the following custom config for the workspace:
```
"eslint.workingDirectories": [
    "brave"
  ],
 ```
 Other than that I don't think I have anything different, and no custom config for the Prettier extension.

Reply:
Sweet, got that working - had to reinstall prettier :D I think there was some file that was synced from a different machine